xbmc/cores/VideoPlayer/VideoRenderers/WinRenderer.cpp

```diff
@@ -1065,6 +1065,9 @@ bool CWinRenderer::Supports(ESCALINGMETHOD method)
 
 EINTERLACEMETHOD CWinRenderer::AutoInterlaceMethod()
 {
+  if (m_renderMethod == RENDER_DXVA)
+    return VS_INTERLACEMETHOD_DXVA_AUTO;
```

Comment:
just return VS_INTERLACEMETHOD_RENDER_BOB

introducing a fake method that is not presented to the user is confusing